Funcky/Monads/Option/Option.Convenience.cs

```diff
@@ -11,43 +11,60 @@ namespace Funcky.Monads
     {
```

Comment:
What happened here?

Reply:
I can't handle Rider :) Fixed in 8d485a9c